### PR TITLE
quadlet systest: fix broken tmpdir references

### PR DESCRIPTION
**Pull request by edsantiago**
quadlet volume-path system test was making invalid assumptions
about $TMPDIR, causing test to fail when TMPDIR=/var/tmp or /dev/shm

Minor cleanup too.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

File: test/system/252-quadlet.bats [hunk 1]
```diff
@@ -762,18 +761,16 @@ EOF
     cat > $quadlet_file <<EOF
 [Container]
 Image=$IMAGE
-Volume=%T/$tmp_dir:/test_content:Z
+Volume=$tmp_path:/test_content:Z
```
Comment (Luap99) [inline test/system/252-quadlet.bats:764]: This breaks the purpose of the test, we have to test `%T`, da96ff6c

**Review by Luap99 — CHANGES_REQUESTED** [review]

File: test/system/252-quadlet.bats [hunk 2]
```diff
@@ -752,8 +752,32 @@ EOF
 }
 
 @test "quadlet - volume path using specifier" {
-    local tmp_path=$(mktemp -d --tmpdir=$PODMAN_TMPDIR quadlet.volume.XXXXXX)
-    local tmp_dir=${tmp_path#/tmp/}
+    # Step 1: determine what systemd is using for %T. There does not
+    # seem to be any systemctly way to find this.
```
Comment (Luap99) [inline test/system/252-quadlet.bats:756]: I think we could try `systemd-run [--user] -q --pipe printenv TMPDIR`
based on `systemd.unit(5)` `%T` defaults to $TMPDIR and if unset /tmp. Not much better but avoids the tmp unit.

Comment (edsantiago) [reply test/system/252-quadlet.bats:756]: I'm not sure that's safe enough for me. From `systemd.unit(5)`, `%T` is:
> either /tmp or the path "$TMPDIR", "$TEMP" or "$TMP" are set to.

**Comment by ygalblum** [comment]
LGTM

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by ygalblum** [comment]
@Luap99 you'll need to approve the PR in order to clear the "Requested Changes" state

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**